src/care/config.h.in

```diff
@@ -30,21 +32,58 @@
 #define CARE_DLL_API
 #endif // defined(_WIN32) && !defined(CARESTATICLIB)
 
+#if defined(ENABLE_CUDA)
+#include <cuda_runtime_api.h>
+#elif defined(ENABLE_HIP)
+#include <hip/hip_runtime.h> // HIPCC only gets defined when we include this. Needed to define GPUCC below
+#endif
+
+#if defined(__CUDA_ARCH__) || defined(__HIP_DEVICE_COMPILE__)
+#define CARE_DEVICE_COMPILE
+#endif
+
 #if defined(__CUDACC__) || defined(__HIPCC__)
-#define __GPUCC__
+#define CARE_GPUCC
 #endif
 
-#if defined(__GPUCC__)
+#if defined(CARE_GPUCC)
 #define CARE_HOST_DEVICE __host__ __device__
 #define CARE_DEVICE __device__
 #define CARE_HOST __host__
 #define CARE_GLOBAL __global__
-#else // defined(__GPUCC__)
+#else // defined(CARE_GPUCC)
 #define CARE_HOST_DEVICE
 #define CARE_DEVICE
 #define CARE_HOST
 #define CARE_GLOBAL
-#endif // defined(__GPUCC__)
+#endif // defined(CARE_GPUCC)
+
+// set various aliases
+#if defined(__CUDACC__)
+
+#define gpuStream_t cudaStream_t
```

Comment:
At some point we'll want to consider using umpire and let it take care of all these things.

Reply:
That could be done in the future, for sure. It would be nice if umpire had all of these defined and then both CHAI, CARE, and other users could all use the same thing